src/SignalR/clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -376,6 +413,83 @@ export class HttpConnection implements IConnection {
         }
     }
 
+    private async reconnect(error?: Error) {
+        try {
+            await this.startPromise;
+        } catch (e) {
+            // This exception is returned to the user as a rejected Promise from the start method
+            // or was observed by the last reconnect loop.
+        }
+
+        const reconnectPolicy = this.options.reconnectPolicy as IReconnectPolicy;
+        let previousRetryCount = 0;
+        let nextRetryDelay = reconnectPolicy.nextRetryDelayInMilliseconds(previousRetryCount++, 0);
+
+        if (nextRetryDelay === null) {
+            this.logger.log(LogLevel.Information, "Connection not reconnecting because of the IReconnectPolicy.");
```

Comment:
There's a weird situation where it's possible to have reconnectPolicy.nextRetryDelayInMilliseconds() return null the first time it's called. That's what this is handling. I don't want to log "Reconnect attempts have stopped" when the HttpConnection never entered the reconnecting state.

That being said, it is possible that the HubConnection never left the reconnecting state from the previous reconnect attempt, and this will be logged. I'll try changing the log based on the previous retry count.